analysis/src/CompletionBackEnd.ml

```diff
@@ -1572,6 +1572,81 @@ let completeTypedValue ~env ~envWhereCompletionStarted ~full ~prefix
   in
   completeTypedValueInner ~env ~full ~prefix ~expandOption
 
+let getJsxLabels ~componentPath ~findTypeOfValue ~package =
```

Comment:
This fn is mostly just extracted for reuse, and adapted so it also returns the relevant `env` for the labels produced.